organization/src/integrationTest/kotlin/com/cosmotech/organization/service/OrganizationServiceIntegrationTest.kt

```diff
@@ -1283,1447 +1283,1502 @@ class OrganizationServiceIntegrationTest : CsmRedisTestBase() {
+      assertThrows<CsmResourceNotFoundException> {
+        val name = "o-connector-test-1"
+        runAsDifferentOrganizationUser()
+        val organizationRegistered =
+            organizationApiService.registerOrganization(
+                createTestOrganizationWithSimpleSecurity(
+                    name, TEST_ADMIN_USER_ID, ROLE_NONE, ROLE_VIEWER))
+        runAsPlatformAdmin()
+        organizationApiService.getOrganizationAccessControl(
+            organizationRegistered.id!!, UNKNOWN_IDENTIFIER)
       }
+    }
 
-      @Test
-      fun `getOrganizationAccessControl with not owned organization, READ_SECURITY permission and existing user`() {
-        assertDoesNotThrow {
-          val name = "o-connector-test-1"
-          runAsDifferentOrganizationUser()
-          val organizationRegistered =
-              organizationApiService.registerOrganization(
-                  createTestOrganizationWithSimpleSecurity(
-                      name, TEST_ADMIN_USER_ID, ROLE_NONE, ROLE_VIEWER))
-          runAsPlatformAdmin()
-          val organizationRole =
-              organizationApiService.getOrganizationAccessControl(
-                  organizationRegistered.id!!, TEST_ADMIN_USER_ID)
-          assertNotNull(organizationRole)
-          assertEquals(ROLE_VIEWER, organizationRole.role)
-          assertEquals(TEST_ADMIN_USER_ID, organizationRole.id)
-        }
+    @Test
+    fun `getOrganizationAccessControl with not owned organization, no READ_SECURITY permission, non-existing user`() {
+      assertThrows<CsmResourceNotFoundException> {
+        val name = "o-connector-test-1"
+        runAsDifferentOrganizationUser()
+        val organizationRegistered =
+            organizationApiService.registerOrganization(
+                createTestOrganizationWithSimpleSecurity(
+                    name, TEST_ADMIN_USER_ID, ROLE_NONE, ROLE_NONE))
+        runAsPlatformAdmin()
+        organizationApiService.getOrganizationAccessControl(
+            organizationRegistered.id!!, UNKNOWN_IDENTIFIER)
       }
+    }
 
-      @Test
-      fun `getOrganizationAccessControl with not owned organization, READ_SECURITY permission and non existing user`() {
-        assertThrows<CsmResourceNotFoundException> {
-          val name = "o-connector-test-1"
-          runAsDifferentOrganizationUser()
-          val organizationRegistered =
-              organizationApiService.registerOrganization(
-                  createTestOrganizationWithSimpleSecurity(
-                      name, TEST_ADMIN_USER_ID, ROLE_NONE, ROLE_VIEWER))
-          runAsPlatformAdmin()
-          organizationApiService.getOrganizationAccessControl(
-              organizationRegistered.id!!, UNKNOWN_IDENTIFIER)
-        }
+    @Test
+    fun `getOrganizationAccessControl with not owned organization, no READ_SECURITY permission, existing user`() {
+      assertDoesNotThrow {
```

Comment:
assertDoesNotThrow should be specific else it's the same as not having it

Reply:
same comment as above